.ci/generate_bear_requirements.py

```diff
@@ -144,6 +154,47 @@ def get_pip_requirements(requirements):
     return _get_requirements(requirements, '~=', inherited_requirements)
 
 
+def deep_update(target, src):
+    for key, value in src.items():
+        if key not in target:
+            target[key] = copy.deepcopy(value)
+        else:
+            if isinstance(value, list):
+                target[key].extend(value)
+            elif isinstance(value, dict):
+                deep_update(target[key], value)
+            elif isinstance(value, set):
+                target[key].update(value.copy())
+            else:
+                target[key] = copy.copy(value)
+
+
+def deep_diff(target, src):
+    errors = []
+    for key, value in src.items():
+        if key not in target:
+                errors.append((key, 'Missing'))
```

Comment:
bad indentation

please run pycodestyle on your code, and still manually review your work before pushing.

Reply:
Hmm, I did run it, it did not report this. Also is there an issue for making bears run on the `.ci` folder? Or should I make one?